File: kotlin/lib/src/main/kotlin/models/AutoConfigOut.kt
// This file is @generated
package com.svix.kotlin.models

import kotlin.time.Instant
import kotlinx.serialization.Serializable

@Serializable
data class AutoConfigOut(
    val createdAt: Instant,
    val token: String,
    /** The AutoConfigSubscription's ID. */
    val id: String,
)

File: javascript/src/models/autoConfigOut.ts
// this file is @generated

export interface AutoConfigOut {
  createdAt: Date;
  token: string;
  /** The AutoConfigSubscription's ID. */
  id: string;
}

export const AutoConfigOutSerializer = {
  _fromJsonObject(object: any): AutoConfigOut {
    return {
      createdAt: new Date(object["createdAt"]),
      token: object["token"],
      id: object["id"],
    };
  },

  _toJsonObject(self: AutoConfigOut): any {
    return {
      createdAt: self.createdAt,
      token: self.token,
      id: self.id,
    };
  },
};
